pkgs/development/python-modules/configargparse/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, lib, buildPythonPackage, fetchurl, pythonPackages }:
+
+buildPythonPackage rec {
+  name = "ConfigArgParse-${version}";
+  version = "0.12.0";
+
+  src = fetchurl {
+    url = "mirror://pypi/C/ConfigArgParse/ConfigArgParse-${version}.tar.gz";
+    sha256 = "0fgkiqh6r3rbkdq3k8c48m85g52k96686rw3a6jg4lcncrkpvk98";
+  };
+
+  # no tests in tarball
+  doCheck = false;
+
+  meta = with stdenv.lib; {
```

Comment:
license, description, maintainer?

Reply:
As this package didn't have any maintainer before, I added myself and also a short description from the repo and the license.